### PR TITLE
Update with latest from Friism

### PR DESCRIPTION
**Pull request by drusellers**
This is the commit that Friism had, I will grant that I don't fully understand the changes but I'm happy to tweak as needed now that we have a means to discuss the changes.

### CODE REVIEW

File: cedar-14/build [hunk 1]
```diff
@@ -7,7 +10,19 @@ fi
...
+=======
+
+CACHE_DIR="/var/cache/mono-$VERSION"
+
+if [ ! -d "$CACHE_DIR" ]; then
+    git clone --recursive --depth 1 --branch mono-$VERSION https://github.com/mono/mono.git $CACHE_DIR
+fi
+
+rsync -r $CACHE_DIR . --exclude .git
+>>>>>>> 98b120d... build in a better way
```
Comment (haf) [inline cedar-14/build]: ?

File: cedar-14/build [hunk 2]
```diff
@@ -43,3 +59,22 @@ echo "Compress buildpack"
 tar -zcf mono-$VERSION.tar.gz mono
 echo "Copy buildpack"
 mv mono-$VERSION.tar.gz /var/cache/mono-$VERSION.tar.gz
+=======
```
Comment (haf) [inline cedar-14/build]: ?

File: cedar-14/build [hunk 3]
```diff
@@ -31,6 +46,7 @@ echo "Run make install"
 make install
 echo "Cd to /app"
 cd /app
+<<<<<<< HEAD
```
Comment (haf) [inline cedar-14/build]: ?

File: cedar-14/build [hunk 4]
```diff
@@ -43,3 +59,22 @@ echo "Compress buildpack"
...
+find mono -name *System.Windows.Forms* -delete
+find mono -name sqlmetal.exe -delete
+
+find mono | xargs strip 2> /dev/null
+
+MONO_ARCHIVE=mono-$VERSION.tar.gz
+tar -zcf $MONO_ARCHIVE mono
+rm -rf /var/out/$MONO_ARCHIVE
+mv $MONO_ARCHIVE /var/out/.
+>>>>>>> 98b120d... build in a better way
```
Comment (haf) [inline cedar-14/build]: ?

Comment (drusellers) [inline cedar-14/build]: Well that's embarassing

**Comment by drusellers** [comment]
@haf embarrassing merge notes removed. :facepalm:

**Comment by ademar** [comment]
Thanks

**Pull request merged**